packages/typespec-python/test/azure/generated/authentication-http-custom/setup.py

```diff
@@ -13,7 +13,7 @@
 
 
 PACKAGE_NAME = "authentication-http-custom"
-PACKAGE_PPRINT_NAME = "Authentication Http Custom Service"
+PACKAGE_PPRINT_NAME = "Authentication Http Custom"
```

Comment:
is this change by design?

Reply:
Yeah, the change is caused by configuration change of `regenerate.ts`